docs/manuals/source/Appendix/Howtos/MigrateABareosCatalogFromMySqlToPostgresql.rst.inc

```diff
@@ -31,10 +31,15 @@ Prepare the new database
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
 Firstly, create a new |postgresql| database as described in :ref:`section-CreateDatabase`.
```

Comment:
```suggestion
Firstly, create a new |postgresql| database as described in :ref:`section-CreateDatabase`, and make sure all daemons are stopped.
```

As far as I understand, the daemons have to be stopped since there is a warning a few steps away from here. But if I follow the `Prepare Bareos Database` before seeing the warning, I can be fooled to start the daemons right after running the scripts. The `start daemons` part that comes after `Preapre Bareos Database` in `IntroductionAndTutorial/InstallingBareos` is a separate part. But the separation between titles is not so obvious, the `start daemons` can seem to be part of Prepare Bareos Databse. I fell into the trap, and other users can too.
So we should just explicitly tell them not to start the daemons.

Reply:
Ok we will bring on top the warning about keeping the bareos-dir daemon stopped all the time of the migration.